SAS/Jonckheere-Terpstra Test.qmd

```diff
@@ -0,0 +1,106 @@
+------------------------------------------------------------------------
```

Comment:
Please rename your qmd file to remove spaces and capital letters

jonchkheere_terpstra.qmd

Reply:
> Please rename your qmd file to remove spaces and capital letters
> 
> jonchkheere_terpstra.qmd

Updated.